actors/datacap/src/types.rs

```diff
@@ -2,6 +2,66 @@ use fvm_ipld_encoding::tuple::*;
 use fvm_shared::address::Address;
 use fvm_shared::econ::TokenAmount;
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct ConstructorParams {
+    pub governor: Address,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct NameReturn {
```

Comment:
Could you add an issue in https://github.com/helix-onchain/filecoin to move these types there?

Reply:
https://github.com/helix-onchain/filecoin/issues/206